### PR TITLE
- #73: Time tracking optional by user

### PR DESCRIPTION
**Pull request by pprolancer**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -15,16 +15,12 @@ <h4 style="text-align:center;">Record Patient Visit </h4>
           <form method="POST" class="form-horizontal" action=".">
           {% csrf_token %}
           {% for field in refform.visible_fields %}
+          {% if field.name != 'visit_appointment_time' and field.name != 'visit_actual_time' %}
+
           <div class="form-group">
```
Comment (pedro-walter) [inline templates/index.html]: This line should appear before the {% if %}.

**Comment by pedro-walter** [comment]
The current code changed only the index page, if you try to add via the form on "Options" -> "Patient Visit" -> "Add", the fields are always available.
Other than my comment above, everything is fine.

File: templates/index.html (hunk 1 above)
Comment (pprolancer) [inline templates/index.html]: Hi @pedro-walter 
regarding "more fields", we dont need to have this option in "Options" -> "Patient Visit" -> "Add" page.
just, can you explain more which line should be appear before the {% if %} ?

**Comment by vincentdavis** [comment]
@pprolancer thanks, I have merged

**Comment by pedro-walter** [comment]
After talking with pprolancer, he is right, the <div> should appear _after_ the {% if %}. So everything was in order.

**Pull request merged**